hw/mcu/dialog/da1469x/src/hal_flash.c

```diff
@@ -307,6 +307,28 @@ da1469x_qspi_erase_sector(const struct hal_flash *dev, uint32_t sector_address)
     return 0;
 }
 
+void
+da1469x_flash_setup(const struct hal_flash *dev, uint32_t bursta,
+                    uint32_t burstb, uint8_t *cmds, int len)
+{
+    int i;
+
+    da1469x_qspi_mode_manual(NULL);
```

Comment:
while da1469x_qspi_mode_manual does not use it's argument for now, it is strange to pass NULL while dev that is in this function is not used.